styles/style.css

```diff
@@ -0,0 +1,187 @@
+body
```

Comment:
Formatting changes:

* Please follow the opening brace in the same line. For example:

```css
.class {
  ...
}
```

* Please add space after a semicolon everywhere. For example it should be `margin: 0` and not `margin:0`.

* Preferably leave one line gap after each rule, for clarity.

Reply:
Will surely look into these in further code.Also will try to change wherever possible.